xml/System.IO.Enumeration/FileSystemEntry.xml

```diff
@@ -23,7 +23,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Provides a lower level view of <see cref="T:System.IO.Enumeration.FileSystemInfo" /> to help process and filter find results.</summary>
```

Comment:
```suggestion
    <summary>Provides a lower level view of <see cref="T:System.IO.FileSystemInfo" /> to help process and filter find results.</summary>
```

Reply:
I've fixed this one already.

Reply:
Broken link

Reply:
I saw your commit, and this was the only one left.

Reply:
Ah, cool! Thanks!